### ISSUE TITLE
Tone down colors on portions of site

### ISSUE THREAD

**Issue by dennisgaebel** [a11yproject/a11yproject.com#964]
Some parts of the site are harsh on the eyes in terms of color. We'll need to list other spots on site that need addressing and pick accessible values that meet contrast guidelines.

This part in particular is one example of a color choice that's not ideal on the eyes.
![Screen Shot 2020-07-16 at 12 07 26 PM](https://user-images.githubusercontent.com/13485205/87695122-1a93c800-c75d-11ea-88ad-8363291974f5.png)

**Comment by dennisgaebel** [comment]
Title bars of these cards get lost with white bg and could use some love to help differentiate them

![Screen Shot 2020-07-16 at 12 12 53 PM](https://user-images.githubusercontent.com/13485205/87695618-b1f91b00-c75d-11ea-90c4-f8f1dba34b49.png)

**Comment by ericwbailey** [comment]
Chiming in here to say that I've received some reports via personal email and DMs about areas where the more bright colors vibrate to the point of discomfort. Notably, white on bright pink is causing Chromostereopsis-style effects.

We should investigate toning down the vibrating effect and determine if we can lessen these effects without moving too far away from Tatiana's vision.

In the spirit of https://github.com/a11yproject/a11yproject.com/issues/939 and honoring user choice, I'd personally like to create a muted color theme. This way we'd offer multiple way for the user to self-select a visual treatment that works for them. 

I especially like how Max Böck does this on their personal website with [the theme chooser](https://mxb.dev/). @maxboeck: Hope you don't mind me tagging you, I noticed your implementation has fantastic keyboard support, and I was wondering if I could trouble you on some insights about how you set yours up?

**Comment by maxboeck** [comment]
@ericwbailey sure - love the work you all are doing with a11yproject!
I did a quick write up of how the theme switcher is put together here: 

https://mxb.dev/blog/color-theme-switcher/

It's basically just a list of buttons, so a lot of the keyboard support just comes naturally with that. 
I'm also using the focus-visible polyfill and have some styling for the active theme, and a bit of JS to manage focus when the switcher panel is opened and closed.

the site is based on 11ty as well, so your implementation could probably be very similar.
all of the code is public, i.e. https://github.com/maxboeck/mxb/blob/master/src/includes/themepicker.njk

let me know if I can help with anything else!

**Comment by ericwbailey** [comment]
Amazing, thank you!

**Comment by dennisgaebel** [comment]
@ericwbailey If y’all want a muted colors theme option then I suggest making this feature 1 issue (Color Theme Switcher). Seems like making a way to control color schemes is what's desired, and since that’s the case it would be more efficient, and easier to understand the goals by having this feature as 1 single issue with the desires listed out, and removing the [dark mode](https://github.com/a11yproject/a11yproject.com/issues/939) issue previously filed.

Also as a side note having schemes that target Protanopia, Deuteranopia etc should be included in this color scheme switcher just like you would find in Slack IMHO 🍻

**Comment by ericwbailey** [comment]
Sounds good to me! Do you want to consolidate the issues, or shall I?

**Comment by dennisgaebel** [comment]
@ericwbailey Go for it! All you. I'll jump in where needed, but I think it's a good idea® to list our your requirements so we can make next steps for whatever needs to be tackled. 👍

**Comment by ericwbailey** [comment]
Closing via https://github.com/a11yproject/a11yproject.com/issues/1008, as discussed in [this comment](https://github.com/a11yproject/a11yproject.com/issues/964#issuecomment-661336876).

@StommePoes, acknowledging your comment before closing. We'll make sure this is addressed.
